modules/tinymce/src/core/main/ts/util/Quirks.ts

```diff
@@ -438,6 +438,16 @@ const Quirks = (editor: Editor): Quirks => {
     });
   };
 
+  /**
```

Comment:
Is this stating the obvious? The function name is pretty clear. So I think the comment is not needed.